### DISCUSSION TITLE
可不可以考虑在版本号之后加上构建日期（或者构建号）？

### DISCUSSION THREAD

**Discussion by SALTWOOD** [Meloong-Git/PCL#3282] [讨论]
感觉这样子对反馈 Bug 有用，因为 PCL2 不是反馈一个 Bug 就立刻提交修复的代码、发布新构建然后新版就没这个 Bug 那种

**Comment by Pigeon0v0** [comment]
没看明白有啥意义...

除了极少数情况外，版本号都是唯一的

**Reply by SALTWOOD** [reply]
意思是加一个时间，这样子方便确认已被标记成“完成”的 Issue 是否被同步至此版本中

**Reply by LTCatt** [reply]
umm 可以去查看 [Release](https://github.com/Hex-Dragon/PCL2/releases/) 的时间？

**Reply by SALTWOOD** [reply]
> umm 可以去查看 [Release](https://github.com/Hex-Dragon/PCL2/releases/) 的时间？

好主意（）我没想到
